cmd/kubeadm/app/cmd/init.go

```diff
@@ -258,6 +258,8 @@ func NewInit(cfgPath string, externalcfg *kubeadmapiv1alpha2.MasterConfiguration
 	if err := preflight.RunInitMasterChecks(utilsexec.New(), cfg, ignorePreflightErrors); err != nil {
 		return nil, err
 	}
+
+	glog.Infoln("[preflight] pulling images required by kubeadm")
```

Comment:
fmt.Println please
also, maybe something like
```
[preflight/images] Pulling images required for setting up a Kubernetes cluster
[preflight/images] This might take a minute or two, depending on the speed of your internet connection.
[preflight/images] You can also perform this action in beforehand using 'kubeadm config images pull'
```